src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,141 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void setUp() {
+        User user1 = new User();
+        user1.setLogin("admin");
+        user1.setPassword("adminPassword");
+        user1.setAge(20);
+        user1.setId(0L);
+        Storage.people.add(user1);
+        User user2 = new User();
+        user2.setLogin("someLogin");
+        user2.setPassword("somePassword");
+        user2.setAge(18);
+        user2.setId(1L);
+        Storage.people.add(user2);
+        User user3 = new User();
+        user3.setLogin("randomLogin");
+        user3.setPassword("randomPassword");
+        user3.setAge(43);
+        user3.setId(2L);
+        Storage.people.add(user3);
+    }
+
+    @Test
+    void register_NotContainsUserLoginInStorage_Ok() {
```

Comment:
The first letter must be lowercase
not "NotContains" -> "register_notContains"
check other method names...